CoEpi/domain/model/CENKey.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import CryptoKit
+import RealmSwift
+import Security
+
+struct CENKey : Codable {
+    var cenKey: String?
+    var timestamp: Int64 = Int64(Date().timeIntervalSince1970)
```

Comment:
Why mutable fields? And the inconsistency with how `CEN` is initialized?

Reply:
Made lets. What do you mean by inconsistency with how CEN is initialized?

Reply:
Aside of `var`, that you use `init` there.

Reply:
Should be consistent now